### DISCUSSION TITLE
Do not succeed installing plugin on Ubuntu / OBS 30.1.2. (Flatpak)

### DISCUSSION THREAD

**Discussion by philippulus** [sorayuki/obs-multi-rtmp#424] [Q&A]
I have downloaded the flatpak file, decompressed it, and copied it in .var/app/com.obsproject.Studio/config/obs-studio/plugins 
I restarted OBS but the plugin does not show up. 
Do I miss something here? 
Is there a proven way to install this plugin in OBS 30.1.2 on Ubuntu (24.04 LTS)?

![Screenshot from 2024-05-25 22-30-53](https://github.com/sorayuki/obs-multi-rtmp/assets/89039695/633f11c4-f587-4440-8e0e-ab2d48a8f58d)

**Comment by sorayuki** [comment]
The script is not used now. I’ll delete it later if it doesn’t work.
The GitHub action script contains a full pipeline to build the plugin, but it’s written towards the GitHub action environment. I’m afraid you’ll have a look into the contents of those script to decide what to do if you want to build it manually.

________________________________
发件人: philippulus ***@***.***>
发送时间: Sunday, May 26, 2024 3:41:07 AM
收件人: sorayuki/obs-multi-rtmp ***@***.***>
抄送: Subscribed ***@***.***>
主题: [sorayuki/obs-multi-rtmp] Do not succeed installing plugin on Ubuntu / OBS 30.1.2. (Flatpak) (Discussion #424)


I have downloaded the code as zip file as explained here.
As the /plugin subfolder was not existing I created manually it at .var/app/com.obsproject.Studio/config/obs-studio/plugins
Then I have unpacked the zip file and moved the obs-multi-rtmp-master folder to this /plugins directory.
I restarted OBS but the plugin does not show up.

Could not create a Flatpak installation package according to this tutorial<https://github.com/sorayuki/obs-multi-rtmp/discussions/282> since the .github / scripts subfolder does not contain a build-linux.sh file but a build-linux file instead.

Trying to install via cmake as proposed here<https://github.com/sorayuki/obs-multi-rtmp/discussions/231> also does not work as I encounter errors with running the code.

Is there a proven way to install this plugin in OBS 30.1.2 on Ubuntu (24.04 LTS)?

―

**Reply by philippulus** [reply]
Thank you for your reply.
I had edited my comment as I had meanwhile found the Flatpak for download. 
I have placed it in the right folder as shown in your table, but it does work any better (pls see first edited post).

**Reply by sorayuki** [reply]
Linux is something I'm not good at. If the plugin loaded successfully, it should be shown in the "Docks" menu

**Comment by philippulus** [comment]
Oh! yes, it is in there. 
Thank you.
